cli/tests/unit/test_marathon_pod.py

```diff
@@ -40,3 +53,28 @@ def resource_reader(path):
         subcmd.pod_add('does/not/matter')
 
     assert exception_info.value == exception
+
+
+def _assert_pod_remove_invoked_successfully(pod_id, force):
+    def resource_reader(path):
+        pass
```

Comment:
instead of passing, can we throw an exception to make sure this isn't doing anything?